CMakeSettings.json

```diff
@@ -121,6 +121,86 @@
           "type": "BOOL"
         }
       ]
-    }
-  ]
+        },
```

Comment:
I will remove this. This is just an illustration how to build with contrib from Visual Studio. I'll move this into separate document.